programs/cleos/CLI11.hpp

```diff
@@ -2434,3 +2434,40 @@ struct AppFriend {
 } // namespace detail
 
 } // namespace CLI
+
+namespace eosio {
+   namespace cleos {
+   struct KeosdSignalHndl {
```

Comment:
Use snake case for all struct/classes. Follow code style found in existing code.